tsconfig.json

```
@@ -22,6 +22,18 @@
       "@types/node",
       "vue-apollo",
       "vitest/globals"
+    ],
+    "plugins": [
+      {
+        "name": "@0no-co/graphqlsp",
+        "schemas": [
+          {
+            "name": "ahp",
+            "schema": "https://ahp.gql.api.kodadot.xyz/",
+            "tadaOutputLocation": "./queries/clients/types/ahp.d.ts"
```

The folder doesn't exist; it won't create the types on launch. Adjust it and ignore `*.d.ts` files

```suggestion
            "tadaOutputLocation": "./queries/clients/ahp.d.ts"
```